app/api/route.js

```diff
@@ -0,0 +1,298 @@
+import Author from "@/models/authors";
+import { ApolloServer } from "@apollo/server";
+import { startServerAndCreateNextHandler } from "@as-integrations/next";
+import DataLoader from "dataloader";
+import { gql } from "graphql-tag";
+import moment from "moment/moment";
+import { validateImageUrl } from "../utils/imageValidator";
+const Book = require("../../models/books");
+const { Op } = require("sequelize");
+
+const typeDefs = gql`
+  type Book {
+    id: Int!
+    title: String!
+    description: String!
+    publishedAt: String!
+    profilePhotoUrl: String
+    author: Author!
+    author_id: Int!
+    reviews: [Review]
+  }
+
+  type Review {
+    rating: Int!
+    comment: String
+  }
+
+  type Author {
+    id: Int!
+    name: String!
+    biography: String!
+    profilePhotoUrl: String
+    books: [Book]
+    date_of_birth: String!
+  }
+
+  type BookConnection {
+    books: [Book!]!
+    totalCount: Int!
+    hasMore: Boolean!
+  }
```

Comment:
How would this change if we want to move both back and forth in terms of page? Currently this will allow only moving to next page and not back right?